library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -309,7 +309,23 @@ public void setLoopCount(int loopCount) {
         }
 
         if (loopCount == LOOP_INTRINSIC) {
-            maxLoopCount = decoder.getLoopCount();
+            int netscapeIterationCount = decoder.getNetscapeIterationCount();
```

Comment:
As far as I see `GifHeader` is the raw, `GifDecoder` is the interpretation and `GifDrawable` is the display of the interpretation. As I said, you can use `GifDecoder` directly to decode frames without using the Glide pipeline.

I would name them `getNetscapeLoopCount()` (to get raw header data) and `getTotalIterationCount()` (to get useful data) so to distinguish between "loop" and "iteration": "iteration" being one repeat of frames, "loop" being a repeated iteration after the first iteration. This should reflect the "loop+1" interpretation in the Chrome issue. You could also emphasize this with some JavaDoc on these methods. I think the mapping between 0 (`NETSCAPE_FOREVER`) and GlideDrawable's one is OK to stay there.